examples/DotNetCore/WebDemo/Microsoft.Azure.AppConfiguration.WebDemo/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using System;
+
+namespace Microsoft.Azure.AppConfiguration.WebDemo
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            BuildWebHost(args).Run();
+        }
+
+        public static IWebHost BuildWebHost(string[] args) =>
+            WebHost.CreateDefaultBuilder(args)
+                .ConfigureAppConfiguration((hostingContext, config) =>
+                {
+                    //
+                    // This example uses the Microsoft.Extensions.Configuration.AzureAppConfiguration NuGet package:
+                    // Loads settings from Azure App Configuration.
+                    // Sets up the provider to listen for changes triggered by a sentinel value.
+                    // Establishes the connection to Azure App Configuration via Managed Identity for Azure Services.
+                    var settings = config.Build();
+
+                    config.AddAzureAppConfiguration(options => {
+
+                        options.ConnectWithManagedIdentity("https://demo-app.azconfig.io")
```

Comment:
>https://demo-app.azconfig.io [](start = 60, length = 28)

We shouldn't have a hardcoded uri. We should have something like this


```
                    IConfiguration settings = config.Build();

                    string appConfigurationEndpoint = settings["AzureAppConfigurationEndpoint"]

                    if (!string.IsNullOrEmpty(appConfigurationEndpoint))
                    {
                        config.AddAzureAppConfiguration...
                        //
                        // lines 26-36 go here
                        //
                    }

```

Reply:
I saw you updated this but you are not checking for `string.IsNullOrEmpty(appConfigurationEndpoint)` like in the example?